services/mailservice/mail_test.go

```diff
@@ -21,9 +21,11 @@ import (
 )
 
 func TestMailConnectionFromConfig(t *testing.T) {
-	cfg, _, _, err := config.LoadConfig("config.json")
+	fs, _, err := config.NewFileStore("config.json", false)
```

Comment:
Needs the actual `config.json` to connect to the `SMTPServer`. Could probably have used a memory store, though, given that we override via environment variables on the CI.